app.ts

```diff
@@ -1,11 +1,10 @@
 import * as express from "express"
 import * as cors from "cors"
+import * as mongoose from "mongoose";
 
 const app = express();
 const port = process.env.PORT || 8080;
 
-const projectRoutes = require("./routes/projectRoutes");
-const mongoose = require("mongoose");
 
 // const dbURI = `mongodb+srv://marta:${hasloDB.haslo}@cluster0.w2bha.mongodb.net/rekruter?retryWrites=true&w=majority`;
```

Comment:
skoro nie korzystamy z bazy mongo to prosę o usunięcie komentarzy
chyba że ostatecznie będziecie  pisać w mongo. Proszę sie określić